toxcore/group_chats.c

```diff
@@ -7034,27 +7035,24 @@ static void do_gc_tcp(const GC_Session *c, GC_Chat *chat, void *userdata)
  * Updates our TCP and UDP connection status and flags a new announcement if our connection has
  * changed and we have either a UDP or TCP connection.
  */
-#define GC_SELF_CONNECTION_CHECK_INTERVAL 2
+#define GC_SELF_CONNECTION_CHECK_INTERVAL 5
```

Comment:
Please add a comment in what unit these intervals are defined. I guess seconds, but I don't know.

Reply:
Done.